### PR TITLE
Increase the default server names length in nginx config file

### PR DESCRIPTION
**Pull request by Sindvero**
It's going to increase the default server names length in the nginx config to avoid the following error:

```
[emerg] 8#8: could not build server_names_hash, you should increase server_names_hash_bucket_size: 64
nginx: [emerg] could not build server_names_hash, you should increase server_names_hash_bucket_size: 64
```

That will allow people with very long domain names (_e.g._ [very.very.long.domain.name.used.in.this.example.org](http://very.very.long.domain.name.used.in.this.example.org/)) to make it work out of the box without having to mount the nginx.conf file in the pod.

Issue #590
And slack discussion: https://pixie-community.slack.com/archives/CQ63KEVFY/p1663087667528489

Reference to change this value:
- https://sleeplessbeastie.eu/2019/11/18/how-to-increase-the-default-number-of-maximum-server-names-and-their-length-when-using-nginx/
- https://gist.github.com/muhammadghazali/6c2b8c80d5528e3118613746e0041263

### CODE REVIEW

**Comment by zasgar** [comment]
ok to test

File: src/cloud/proxy/nginx.conf [hunk 1]
```diff
@@ -39,6 +39,9 @@ http {
     # Tune nginx keepalives to work with the GCP HTTP(S) Load Balancer:
     keepalive_timeout 650;
     keepalive_requests 10000;
+  
+    # Increase default server names lenght to 128:
```
Comment (zasgar) [inline src/cloud/proxy/nginx.conf:43]: nit: can you fix the typo lenght -> "length"? It's triggering one of our lint rules. Thanks!

**Review by zasgar — CHANGES_REQUESTED** [review]

File: src/cloud/proxy/nginx.conf (hunk 1 above)
Comment (Sindvero) [reply src/cloud/proxy/nginx.conf]: Fixed it. And looks like your Jenkins build also passed

**Comment by zasgar** [comment]
ok to test

**Review by zasgar — APPROVED** [review]

**Pull request closed without merging**